### PR TITLE
As a user I would like to create volumes [ch1868]

### PR DESCRIPTION
**Pull request by jlaswell**

### CODE REVIEW

**Comment by kyleferguson** [comment]
@jlaswell this all looks good, only thing is volumes I don't think need to be a "separate" resource. Volumes, I think, are more of a configuration option for versions. Just like we store container params as apart of the version and not as a separate resource.

When you create versions or eventually update you would simply specify whatever volumes need to be created and/or modified at that time. And since we're storing the deployment spec JSON with a version, it will actually already be persisted in that JSON if the deployment has the volumes specified.

File: app/app.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func New() *Application {
 		&resources.Application{},
 		&resources.Environment{},
 		&resources.Version{},
+		&resources.Volume{},
```
Comment (jlaswell) [inline app/app.go]: removing this

**Comment by kyleferguson** [comment]
Nice work on all of this John!

Resolving conflicts my work has with this on my branch. Don't merge this or https://github.com/generationtux/brizo/pull/64 until a path forward has been determined.

**Comment by kyleferguson** [comment]
This is merged with https://github.com/generationtux/brizo/pull/64

**Pull request closed without merging**